tests/unit/pages/quiz/Quiz.spec.jsx

```diff
@@ -0,0 +1,87 @@
+import { screen, fireEvent } from '@testing-library/react'
+import Quiz from '~/pages/quiz/Quiz'
+import useAxios from '~/hooks/use-axios'
+import { renderWithProviders } from '~tests/test-utils'
+import { act } from 'react-dom/test-utils'
```

Comment:
Maybe we should import `act` from `@testing-library/react`

Reply:
Yeap, you are right, I didn't notice it.